api/Makefile

```diff
@@ -107,12 +107,15 @@ create-docker-test-db:
 run: version-file ## Run a development server (reloading)
 	$(RUNNER) flask run -p $(PORT)
 
+newrelic.ini:
+	$(RUNNER) newrelic-admin generate-config $(NEW_RELIC_LICENSE_KEY) newrelic.ini
```

Comment:
Do they happen to have a default for the ini file or do we have to specify everywhere `newrelic.ini`?

Reply:
There doesn't seem to be a default, without the file name it just prints it out to stdout.